it/test/uk/gov/hmrc/emailverification/controllers/EmailVerificationISpec.scala

```diff
@@ -59,49 +66,49 @@ class EmailVerificationISpec extends BaseISpec {
 
     forAll(continueUrls) { (email, template, url) =>
       s"send the verification email to the specified address successfully with continue url $url" in new Setup {
-        Given("The email service is running")
+        // Given("The email service is running")
```

Comment:
Might make sense to remove all of these? I feel like the code is self-describing anyway and the test description covers what is happening?

Reply:
agree, forgot to tidy up